week-03/main.js

```diff
@@ -42,11 +50,34 @@ var list = [1, 2, 3, 4, 5];
 // Set the answer to #answer in the DOM.
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+var odd = function (nums) {
+  return nums % 2 === 0;
+};
+var double = function(num) {
+  return num * 2;
+}
+var adding = function(a, b) {
+    return a + b;
+  }
+
+var filtered = list.filter(odd);
+var doubledUp = filtered.map(double);
+var addedUp = doubledUp.reduce(adding);
 
 //Scope
 
-/* Create a variable in the global scope called name, then write a function called thisName, 
-   which creates another variable inside the function called name, and returns name. What do 
-   you think the output would be when you invoke the function thisName? What happens to the variable 
+/* Create a variable in the global scope called name, then write a function called thisName,
+   which creates another variable inside the function called name, and returns name. What do
+   you think the output would be when you invoke the function thisName? What happens to the variable
    name in the global scope?
 */
+
+var name = "Brad";
+
+var thisName = function(name) {
+  this.name = name;
```

Comment:
`this` sets name on the window. You just want to define a local variable to `name`